### PR TITLE
typos/cleanup in prep for v1.0.1

### PR DESCRIPTION
**Pull request by duglin**
Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,11 +5,22 @@ verify:
...
+		avro-format.md \
+		http-protocol-binding.md \
+		http-webhook.md \
+		json-format.md \
+		kafka-protocol-binding.md \
+		mqtt-protocol-binding.md \
+		nats-protocol-binding.md \
+		protobuf-format.md \
+		spec.md \
+		websockets-protocol-binding.md \
```
Comment (duglin) [inline Makefile:20]: we were missing some files - a single list like this is now easier to track

File: README.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ The following documents are available:
 | MQTT Protocol Binding         | [v1.0](https://github.com/cloudevents/spec/blob/v1.0/mqtt-protocol-binding.md)  |    [master](https://github.com/cloudevents/spec/blob/master/mqtt-protocol-binding.md)    |
 | NATS Protocol Binding         | [v1.0](https://github.com/cloudevents/spec/blob/v1.0/nats-protocol-binding.md)  |    [master](https://github.com/cloudevents/spec/blob/master/nats-protocol-binding.md)    |
 | WebSockets Protocol Binding   |                                        -                                        | [master](https://github.com/cloudevents/spec/blob/master/websockets-protocol-binding.md) |
-| Protobuf Event Format         |                                                                                 |                                 [master][proto-working]                                  |
+| Protobuf Event Format         |                                                                                 | [master](https://github.com/cloudevents/spec/blob/master/protobuf-format.md)                                  |
```
Comment (duglin) [inline README.md:41]: just to make it consistent with the other files pointers

File: community/SDK-GOVERNANCE.md [hunk 3]
```diff
@@ -29,9 +29,9 @@ projects. The purpose of these requirements are to demonstrate the person's
 expertise and regular commitment to the project - not simply to achieve a
 certain level of activity.
 
-Each sdk project MAY define in the `CONTRIBUTING.md` (available at the root of
-the Github Repository) stricter requirements, in order to meet the community
-demands, e.g. _we require that the contributor submitted at least 20 PRs_.
+Each sdk project may define in their `CONTRIBUTING.md` file stricter
+requirements, in order to meet the community demands, e.g. _we require that
+the contributor submitted at least 20 PRs_.
```
Comment (duglin) [inline community/SDK-GOVERNANCE.md:34]: Just wanted to make it clear that we're talking about the CONTRIBUTING in the sdk repo and not the main repo

File: community/SDK-GOVERNANCE.md [hunk 4]
```diff
@@ -80,7 +80,7 @@ discussions". However, if the SDK is stable and does not need to be update then
 it might be determined that it current state is acceptable.
 
 To prevent the project from becoming _not actively maintained_, the community
-MAY takes the following actions:
+may takes the following actions:
```
Comment (duglin) [inline community/SDK-GOVERNANCE.md:83]: This isn't a normative doc so I removed the caps on the RFC2119 keywords

File: community/open-source.md [hunk 5]
```diff
@@ -44,4 +44,4 @@ CloudEvents, please include in the list below.
 - [cloudevent.js](https://github.com/smartiniOnGitHub/cloudevent.js): A
   CloudEvents library for Node.js.
 - [CloudEvents JSON Schema](https://marketplace.visualstudio.com/items?itemName=tsurdilovic.cloudevents-schema-vscode):
-Visual Studio Code extension for CloudEvents format.
+  Visual Studio Code extension for CloudEvents format.
```
Comment (duglin) [inline community/open-source.md:47]: will run 'pretty' on all docs after the review

File: community/SDK-GOVERNANCE.md (hunk 4 above)
Comment (dazuma) [inline community/SDK-GOVERNANCE.md:83]: ...the community may **take** the following...

File: community/SDK-GOVERNANCE.md [hunk 6]
```diff
@@ -108,7 +108,7 @@ enhancements.
 
 ### Handover to a new maintainer/group of maintainers
 
-If a project is not meeting the criteria 1, 2, 4, 5, 6, 7, the community MAY
+If a project is not meeting the criteria 1, 2, 4, 5, 6, 7, the community may
```
Comment (dazuma) [inline community/SDK-GOVERNANCE.md:111]: ...the community may **decide** to **hand over** the project...

File: community/SDK-GOVERNANCE.md [hunk 7]
```diff
@@ -140,8 +140,8 @@ The voting criteria are:
 
 ### Archive a project
 
-If a project is not following the criteria 1, 2, 4, 5, 6, 7, the community MAY
-decides to archive the project. Prior to archiving, the community SHOULD first
+If a project is not following the criteria 1, 2, 4, 5, 6, 7, the community may
+decides to archive the project. Prior to archiving, the community should first
```
Comment (dazuma) [inline community/SDK-GOVERNANCE.md:144]: ...the community may **decide** to archive...

**Review by dazuma — CHANGES_REQUESTED** [review]

**Comment by duglin** [comment]
I had to fix some RFC2119 keywords in the protobuf spec.

**Review by markpeek — APPROVED** [review]
LGTM

**Comment by duglin** [comment]
thank @markpeek. Will merge once travis is ok (if it ever runs)

**Comment by duglin** [comment]
finally got all checks to pass now - and we're using github actions instead of travis
another lgtm would be good :-)

**Comment by dazuma** [comment]
@duglin Do you want to address the grammatical corrections I noted?

File: Makefile [hunk 8]
```diff
@@ -5,11 +5,22 @@ verify:
 	@# Use "-x" if you want to skip external links
 	@tools/verify-links.sh -t -v .
 	@echo Running the spec phrase checker:
-	@tools/verify-specs.sh -v spec.md documented-extensions.md json-format.md \
-		http-protocol-binding.md http-webhook.md mqtt-protocol-binding.md \
-		nats-protocol-binding.md kafka-protocol-binding.md avro-format.md \
+	@tools/verify-specs.sh -v \
+		documented-extensions.md \
```
Comment (markpeek) [inline Makefile:9]: should alphabetize this list (or would it be better to just use "*.md"?

**Review by markpeek — APPROVED** [review]
Just one comment to fix up and then good to go...

File: Makefile (hunk 8 above)
Comment (duglin) [reply Makefile:9]: doit - they were supposed to be :-)
fixed

Comment (duglin) [reply Makefile]: good catch

**Comment by duglin** [comment]
Approved on the 12/3 call.
Made some minor tweaks to make the verification scripts happy - just syntax stuff.

File: community/SDK-GOVERNANCE.md [hunk 9]
```diff
@@ -29,9 +29,9 @@ projects. The purpose of these requirements are to demonstrate the person's
 expertise and regular commitment to the project - not simply to achieve a
 certain level of activity.
 
-Each sdk project MAY define in the `CONTRIBUTING.md` (available at the root of
-the Github Repository) stricter requirements, in order to meet the community
-demands, e.g. _we require that the contributor submitted at least 20 PRs_.
+Each sdk project may define in their `CONTRIBUTING.md` file stricter
```
[the remark this answers is not in the document]
Comment (duglin) [reply community/SDK-GOVERNANCE.md:32]: Fixing in a new PR

File: community/SDK-GOVERNANCE.md (hunk 4 above)
Comment (duglin) [reply community/SDK-GOVERNANCE.md:83]: in next PR

File: community/SDK-GOVERNANCE.md (hunk 6 above)
Comment (duglin) [reply community/SDK-GOVERNANCE.md:111]: next PR

File: community/SDK-GOVERNANCE.md (hunk 7 above)
Comment (duglin) [reply community/SDK-GOVERNANCE.md:144]: next PR

**Comment by duglin** [comment]
> @duglin Do you want to address the grammatical corrections I noted?

So sorry. I was so focused on trying to get the github actions stuff working that I forgot about your comments. I'll get them in a new PR - thanks for the thorough review!

**Pull request merged**